### PR TITLE
Updating the prereqs for submariner

### PR DESCRIPTION
**Pull request by dockerymick**
https://app.zenhub.com/workspaces/engineering-backlog-do-not-delete-604fab62d4b98d00150a2854/issues/stolostron/backlog/21371

### CODE REVIEW

File: add-ons/submariner/submariner.adoc [hunk 1]
```diff
@@ -39,7 +39,13 @@ Ensure that you have the following prerequisites before using Submariner:
 * Firewall configuration open for 4500/UDP and any other ports that are used for IPsec traffic on the gateway nodes.
 +
 *Note:* This is configured automatically when your clusters are deployed in an AWS or GCP environment, but must be configured manually for clusters on other environments and for the firewalls that protect private clouds.
+* The `managedcluster` name must follow the DNS label standard as defined in RFC 1123. This means the name must meet the following criteria:
 +
+- Contain at most 63 characters
+- Contain only lowercase alphanumeric characters or '-'
+- Start with an alphanumeric character
+- End with an alphanumeric character"
```
Comment (Jaanki) [inline add-ons/submariner/submariner.adoc]:
```suggestion
- End with an alphanumeric character
```

Comment (Jaanki) [reply add-ons/submariner/submariner.adoc]: Looks like a left-over.

**Review by Jaanki — CHANGES_REQUESTED** [review]

**Comment by dockerymick** [comment]
@Jaanki thanks for your commit and review! I accepted your suggestion. From my viewpoint, the update looks good. Moving forward with the merge

**Comment by Jaanki** [comment]
Thanks @dockerymick for seeing this through.

**Pull request merged**